config/kernel-bio-op.m4

```diff
@@ -69,6 +69,7 @@ AC_DEFUN([ZFS_AC_KERNEL_BIO_BI_OPF], [
 AC_DEFUN([ZFS_AC_KERNEL_HAVE_BIO_SET_OP_ATTRS], [
 	AC_MSG_CHECKING([whether bio_set_op_attrs is available])
 	ZFS_LINUX_TRY_COMPILE([
+		#include <linux/bio.h>
 		#include <linux/blk_types.h>
```

Comment:
Since `linux/bio.h` includes `linux/blk_types.h` it looks like you can remove this second include as a simplification.